### PR TITLE
Maximezmt/UI

### PR DESCRIPTION
**Pull request by MaximeZmt**
# Coherent UI
### This is related to issue #116 
This Pull Requests contains:

ch.sdp.vibester.activity: GameEndingActivity: Update UI
ch.sdp.vibester.activity: GameSetupActivity: Update UI + add submenu
ch.sdp.vibester.activity: GamescreenActivity: Update UI
ch.sdp.vibester.activity: IncorrectSongActivity: Update UI
ch.sdp.vibester.activity: ScoreboardActivity: Update UI
ch.sdp.vibester.activity: ProfileActivity: Update UI + Correct Image
ch.sdp.vibester.activity: SettingsActivity: Update UI
Readme: has been modified
Logo: Has been refactored

In addition the code is tested with a coverage of: [100]%

</img></img></img></img></img></img></img></img>

### Notes for reviewers
- NONE

### CODE REVIEW

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt [hunk 1]
```diff
@@ -123,6 +135,15 @@ class ProfileActivity : AppCompatActivity() {
         findViewById<TextView>(R.id.ranking).text = user.ranking.toString()
         /* TODO: add functionality to display the image (may be using )
```
Comment (jiabaow) [inline app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:136]:
```suggestion
        

```

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt [hunk 2]
```diff
@@ -123,6 +135,15 @@ class ProfileActivity : AppCompatActivity() {
         findViewById<TextView>(R.id.ranking).text = user.ranking.toString()
         /* TODO: add functionality to display the image (may be using )
         findViewById<ImageView>(R.id.avatar).loadImg(user.image)*/
```
Comment (jiabaow) [inline app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:137]:
```suggestion
   

```

**Review by jiabaow — APPROVED** [review]
LGTM

**Review by kamilababayeva — APPROVED** [review]
The app looks pretty now! Thank you for your work!

Github does not allow me to view all the changed files, so I will leave some small comments there. I think we can fix them in the next PR.

WelcomeActivity:button play music is not working and the app crashes. We need to remove switchToListen function from there.
GameSetupActivity: I liked it more when there was a small arrow in the dropdown for number of players. Now it is not clear if it is clickable or not.
[Question] is it okay that github build badge says failing?

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt [hunk 3]
```diff
@@ -28,6 +36,10 @@ class ProfileActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        requestWindowFeature(Window.FEATURE_NO_TITLE)
+        supportActionBar?.hide()
+
```
[the remark this answers is not in the document]
Comment (MaximeZmt) [reply app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:42]: I don't know if it is doable from the XML, maybe we should look for it next week :)

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt [hunk 4]
```diff
@@ -123,6 +135,15 @@ class ProfileActivity : AppCompatActivity() {
...
         findViewById<ImageView>(R.id.avatar).loadImg(user.image)*/
+        CoroutineScope(Dispatchers.Main).launch {
+            val task = async(Dispatchers.IO) {
+                val bit = BitmapGetterApi.download("https://"+user.image)
+                bit.get()
+            }
+            val bm = task.await()
+            findViewById<ImageView>(R.id.avatar).setImageBitmap(bm)
+        }
+        findViewById<ImageView>(R.id.avatar).foregroundGravity= Gravity.LEFT
```
[the remark this answers is not in the document]
Comment (MaximeZmt) [reply app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:146]: yep will do it

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (MaximeZmt) [reply app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:137]: 👍

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt (hunk 4 above)
[the remark this answers is not in the document]
Comment (MaximeZmt) [reply app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:146]: 👍

File: app/src/main/java/ch/sdp/vibester/activity/GameSetupActivity.kt [hunk 5]
```diff
@@ -85,4 +95,15 @@ class GameSetupActivity : AppCompatActivity(), AdapterView.OnItemSelectedListene
         intent.putExtra("Player Names", pNameArray)
         startActivity(intent)
     }
+
+    private fun choosingGameSetupListener(){
```
[the remark this answers is not in the document]
Comment (MaximeZmt) [reply app/src/main/java/ch/sdp/vibester/activity/GameSetupActivity.kt:99]: 👍

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt (hunk 1 above)
Comment (MaximeZmt) [reply app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:136]: 👍

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt (hunk 2 above)
Comment (MaximeZmt) [reply app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:137]: 👍

**Comment by MaximeZmt** [comment]
> The app looks pretty now! Thank you for your work!
> 
> Github does not allow me to view all the changed files, so I will leave some small comments there. I think we can fix them in the next PR.
> 
> WelcomeActivity:button play music is not working and the app crashes. We need to remove switchToListen function from there. GameSetupActivity: I liked it more when there was a small arrow in the dropdown for number of players. Now it is not clear if it is clickable or not. [Question] is it okay that github build badge says failing?

Concerning the badges that fails, it is normal -> CI has failed and therefore it shows fail, just need a rebuild (would be done once merged)
Yes I will hide the button, I've keeped it temporarily if needed in the future
Yes I will see How I can change it :)

**Pull request merged**